app/Models/Sale.php

```diff
@@ -6,6 +6,7 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\MorphToMany;
 use Illuminate\Support\Facades\Hash;
+use PhpParser\Builder;
```

Comment:
This is the wrong instance of builder it should be `Illuminate\Database\Eloquent\Builder`